changelog.txt

```diff
@@ -1,3 +1,37 @@
+1.11.0-RC1 (2021-03-14)
+------------------------------------------------------------------------
+Feature: Option to (dis-)allow accelerated video drivers (#8819)
+Feature: Display refresh rate game option (#8813)
+Feature: Allow custom width/height of screenshot and making heightmap screenshots via console (#8804)
+Feature: Allow filtering on name in rail station window (#8706)
+Feature: Allow setting maximum zoom level at which sprites are drawn (#8604)
```

Comment:
Tricky thing to summarize to a single line, but still feels a bit off to me
Something like "Add setting to define maximum sprite zoom level", maybe?

Reply:
Went with "Setting for highest resolution of sprites to use", as that is how the setting is called ingame :) This really has been debated way too much :D :D Funny, how difficult some things are to describe in a single sentence :)